controlplane/src/db/schema.ts

```diff
@@ -595,9 +595,93 @@ export const organizations = pgTable('organizations', {
     .references(() => users.id)
     .notNull(),
   createdAt: timestamp('created_at', { withTimezone: true }).notNull().defaultNow(),
-  isPersonal: boolean('is_personal').default(false),
-  isFreeTrial: boolean('is_free_trial').default(false),
-  isRBACEnabled: boolean('is_rbac_enabled').notNull().default(false),
+  /**
+   * @deprecated
+   */
+  isPersonal: boolean('is_personal'),
+  /**
+   * @deprecated
+   */
+  isFreeTrial: boolean('is_free_trial'),
+  /**
+   * @deprecated
+   */
+  isRBACEnabled: boolean('is_rbac_enabled'),
+});
+
+export const organizationBilling = pgTable(
+  'organization_billing',
+  {
+    id: uuid('id').notNull().primaryKey().defaultRandom(),
+    organizationId: uuid('organization_id')
+      .notNull()
+      .references(() => organizations.id, {
+        onDelete: 'cascade',
+      }),
+    plan: text('plan'),
+    email: text('email'),
+    stripeCustomerId: text('stripe_customer_id'),
+  },
+  (t) => {
+    return {
+      orgIndex: uniqueIndex('organization_billing_idx').on(t.organizationId),
+      stripeIndex: uniqueIndex('organization_billing_stripe_idx').on(t.stripeCustomerId),
+    };
+  },
+);
+
+export type Feature = {
+  id: string;
+  description?: string;
+  limit?: number;
+};
+
+export const billingPlans = pgTable('billing_plans', {
+  id: text('id').notNull().primaryKey(),
+  active: boolean('active').notNull().default(true),
```

Comment:
Is it correct to set it active by default? I'd expect it will be activated by a webhook, no?

Reply:
These are just configs for available plans, this is something that's synced with stripe.

Reply:
ok